### ISSUE TITLE
Imagenet download instructions are outdated

### ISSUE THREAD

**Issue by senarvi** [Lightning-AI/lightning-bolts#906]
It would be great to have better instructions on how to download the Imagenet dataset, or at least how to check that you have the correct data package. The [instructions](https://github.com/Lightning-AI/lightning-bolts/blob/master/pl_bolts/datamodules/imagenet_datamodule.py#L133) point to a broken link. The [Imagenet download page](https://www.image-net.org/download.php) instructs to use Kaggle to download the 2012-2017 dataset, but is this the correct subset? At least the directory structure is not correct. According to the [instructions](https://github.com/Lightning-AI/lightning-bolts/blob/master/pl_bolts/datamodules/imagenet_datamodule.py#L134), the devkit also needs to be downloaded, but from where?

**Comment by NeoKish** [comment]
Please do checkout this link
https://image-net.org/challenges/LSVRC/2012/2012-downloads.php#Images

**Comment by senarvi** [comment]
@NeoKish it looks like that link might be the best way to go. The link in the source code led me to download it using Kaggle, and then I found a script that puts the files in correct places, and the development kit I found from another repository, I don't remember where. Then I created `meta.bin` using the code in the data module and copied it to both `train` and `val` directories. So now I have a working dataset that looks like this:

```
train/meta.bin
train/n01440764
...
train/n15075141
val/meta.bin
val/n01440764
...
val/n15075141
```

Under `train` there are 1281167 images and under `val` there are 50000 images in 1000 directories. Not sure if this is the same as LSVRC 2012 though.
